Example/Tests/InMemoryStorage.swift

```diff
@@ -7,19 +7,42 @@
 //
 
 import Foundation
-import Pelican
+@testable import Pelican
```

Comment:
The storable interface only deals with `Data` now so it shouldn't be. I've moved the subscript helpers to `PelicanTests.swift` since that was the only place using them.

Reply:
Is there any way to make `InMemoryStorage` not use `@testable import Pelican`? I would like to have that as an assurance that future changes do not make it impossible for consumers to define their own `PelicanStorage` types.

Reply:
Yes will push up a fixup shortly